gcalcli/gcalcli.py

```diff
@@ -1619,10 +1621,27 @@ def run_add_prompt(parsed_args, printer):
             parsed_args.reminders.append(str(n) + ' ' + m)
 
 
+def _warn_deprecated_opts(argv):
```

Comment:
The issue with this implementation is that it's not robust to = syntax: `--color_owner=yellow` won't be in `ALL_DEPRECATED_OPTS`.

Reply:
We could do something with `.startswtih`, I guess.  That's not very elegant but this code will be excised in a future version, so I may be able to cope with that.